images/google-io/2x.svg

```diff
@@ -0,0 +1 @@
+<svg width="188" height="188" viewBox="0 0 188 133" xmlns="http://www.w3.org/2000/svg"><path fill="#5471F9" d="M0 111.963h47.71V17.9H0z"/><path fill="#31E8B7" d="M82.85 0L53.308 131.147l7.42 1.642L90.273 1.642z"/><path fill="#5471F9" d="M136.648 14c-28.3 0-51.242 22.745-51.242 50.8 0 28.057 22.942 50.8 51.242 50.8 28.301 0 51.242-22.743 51.242-50.8 0-28.055-22.94-50.8-51.242-50.8"/></svg>
```

Comment:
Why are there two SVGs? 🤔

Reply:
Initially I just had one, but then I decided to make it copy it to both so that it is consistent with the other images.